minimal.js

```diff
@@ -95,7 +95,15 @@ module.exports = function (dirname, keys, opts) {
 
   var log = OffsetLog(path.join(dirname, 'log.offset'), { blockSize: 1024 * 16, codec })
 
-  const unboxerMap = (msg, cb) => cb(null, db.unbox(msg))
+  const unboxerMap = (msg, cb) => {
+    db.unbox(msg, null, (err, unboxed) => {
+      if (err) {
+        console.error(err)
+        return cb(null, msg)
+      }
```

Comment:
I'd like input thinking about how to handle error message with boxing / unboxing.

Reply:
This here allows errors to bubble up through callbacks but then pops a log out and "returns" an unecrypted message.